### PR TITLE
completed-react-state-Bishan-Webeshete

### PR DESCRIPTION
**Pull request by BishanWebeshete**
completed react-state

### CODE REVIEW

File: react-state/my-app/src/ToggleButton.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { useState } from 'react';
+export default function ToggleButton({text, color}) {
+  const [isClicked, setIsClicked] = useState(false)
+  function handleClick(){
+    setIsClicked(!isClicked);
+    console.log(isClicked);
+  }
+  if (!isClicked) {
```
Comment (RobertGardner) [inline react-state/my-app/src/ToggleButton.js:8]: This works, but there is quite a bit of duplicated code. The only difference between the two is the color.

Try to figure out how to do this without the duplicated code.

**Review by RobertGardner — CHANGES_REQUESTED** [review]
I'd like you to figure out the duplicated code before I approve.

**Comment by BishanWebeshete** [comment]
duplicate code has been removed

File: react-state/my-app/src/ToggleButton.js [hunk 2]
```diff
@@ -5,11 +5,7 @@ export default function ToggleButton({text, color}) {
     setIsClicked(!isClicked);
     console.log(isClicked);
   }
-  if (!isClicked) {
     return (
-      <button onClick={handleClick} style={{ backgroundColor: 'white' }}>{text}</button>
+      <button onClick={handleClick} style={!isClicked ? { backgroundColor: 'white' } : {backgroundColor: color}}>{text}</button>
```
Comment (RobertGardner) [inline react-state/my-app/src/ToggleButton.js:9]: Good. More succinctly:
`style={{ backgroundColor: isClicked ? color : 'white' }}`

**Review by RobertGardner — APPROVED** [review]

**Pull request merged**